src/api/context.ts

```diff
@@ -49,11 +49,6 @@ export namespace ChectlContext {
       ctx.listrOptions = { renderer: (flags['listr-renderer'] as any), collapse: false } as Listr.ListrOptions
     }
 
-    ctx.operatorNamespace = flags.chenamespace || DEFAULT_CHE_NAMESPACE
```

Comment:
I think we relay on this context field in some places and expect that it is already present. Please check all the places, including preupdate and update tasks.